test/tasks_fine_tuning_task_test.py

```diff
@@ -40,21 +48,18 @@ def test_prepare(self):
         pre_train_task.prepare()
         checkpoint = get_checkpoint_dict(pre_train_task, {})
 
-        fine_tuning_config = self._get_fine_tuning_config()
+        fine_tuning_config = self._get_fine_tuning_config(pretrained_checkpoint=True)
         fine_tuning_task = build_task(fine_tuning_config)
-        # cannot prepare a fine tuning task without a pre training checkpoint
+
         with self.assertRaises(Exception):
             fine_tuning_task.prepare()
 
-        fine_tuning_task.set_pretrained_checkpoint(checkpoint)
-        fine_tuning_task.prepare()
```

Comment:
Shouldn't we keep this? We can pass pretrained_checkpoint=False in line 51 so that lines 49 and 50 would still work, no? Why did you modify this test specifically?

Reply:
You're right, I'll do that